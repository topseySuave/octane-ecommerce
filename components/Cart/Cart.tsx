import { Typography } from "antd";
import { ProductCardHorizontal } from 'components/ui/ProductCard';

const { Title } = Typography;

const Cart = () => {
  return (
    <>
      <Title level={3}>Saved Item(s)</Title>
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
    </>
  );
};

export default Cart;
