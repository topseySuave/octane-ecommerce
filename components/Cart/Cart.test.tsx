import React from 'react';
import renderer from 'react-test-renderer';

import Cart from './Cart';

describe('Cart View', () => {
  it('Renders with horizontal card only', () => {
    const tree = renderer.create(<Cart />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
