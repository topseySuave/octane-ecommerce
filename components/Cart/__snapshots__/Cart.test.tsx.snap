// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart View Renders with horizontal card only 1`] = `
Array [
  <h3
    aria-label={null}
    className="ant-typography"
    style={
      Object {
        "WebkitLineClamp": null,
      }
    }
  >
    Saved Item(s)
  </h3>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
  <div
    className="ant-card oct-horizontal-card ant-card-bordered"
    style={
      Object {
        "marginTop": 18,
      }
    }
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-card-meta"
      >
        <div
          className="ant-card-meta-avatar"
        >
          <span
            className="ant-avatar ant-avatar-circle ant-avatar-image"
            style={Object {}}
          >
            <img
              onError={[Function]}
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
          </span>
        </div>
        <div
          className="ant-card-meta-detail"
        >
          <div
            className="ant-card-meta-title"
          >
            Card title
          </div>
          <div
            className="ant-card-meta-description"
          >
            This is the description
          </div>
        </div>
      </div>
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      >
        <span>
          Remove
        </span>
      </button>
    </div>
  </div>,
]
`;
